DataFormats/Reconstruction/src/TrackParametrizationWithError.cxx

```diff
@@ -1188,11 +1188,23 @@ template <typename value_T>
 std::string TrackParametrizationWithError<value_T>::asString() const
 {
   return TrackParametrization<value_t>::asString() +
-         fmt::format(" Cov: [{:+.3e}] [{:+.3e} {:+.3e}] [{:+.3e} {:+.3e} {:+.3e}] [{:+.3e} {:+.3e} {:+.3e} {:+.3e}] [{:+.3e} {:+.3e} {:+.3e} {:+.3e} {:+.3e}]",
+         fmt::format(" \nCovMat: [{:+.3e}]\n        [{:+.3e} {:+.3e}]\n        [{:+.3e} {:+.3e} {:+.3e}]\n        [{:+.3e} {:+.3e} {:+.3e} {:+.3e}]\n        [{:+.3e} {:+.3e} {:+.3e} {:+.3e} {:+.3e}]",
```

Comment:
I changed it to make it visually look like the already existing implementation: https://github.com/AliceO2Group/AliceO2/blob/b75034eef1914037d963e2bbb5cc00d1810d038d/DataFormats/Reconstruction/src/TrackParametrizationWithError.cxx#L1200

I can do the other way around; what matters to me is being able to do a `diff` of two logs.
Surely this would reduce lines of logs.